sktime/forecasting/base/_fh.py

```diff
@@ -153,12 +204,16 @@ class ForecastingHorizon:
         - if None, the flag is determined automatically:
             relative, if values are of supported relative index type
             absolute, if not relative and values of supported absolute index type
+    freq : str, pd.Index, pandas offset, or BaseForecaster, optional (default=None)
```

Comment:
what do you mean `BaseForecaster`? is it a mistake?

Reply:
meant that it can be a forecaster, in which case the `freq` will be obtained from the `cutoff` stored. Clarified the docstring